code/modules/surgery/organs/lungs.dm

```diff
@@ -0,0 +1,309 @@
+/obj/item/organ/internal/lungs
+	name = "lungs"
+	icon_state = "lungs"
+	parent_organ = "chest"
+	slot = "lungs"
+	organ_tag = "lungs"
+	gender = PLURAL
+	w_class = WEIGHT_CLASS_NORMAL
```

Comment:
It's just a sack of air, pretty sure you can compress it fairly well, but yeah, no stuffing lungs inside pockets